### ISSUE TITLE
Items that deal damage constantly have to be re-equipped to be affected by Torture Pipe

### ISSUE THREAD

**Issue by Draze22** [arcadia-redux/custom_hero_clash_issues#2132]
Basically any passive DOT item (including neutral items) such as Radiance, Cloak of Flames, etc. Items that deal DOT from actives (Fallen Sky) or have durations (Orb of Venom) don't have this issue.

**Comment by ZLOY5** [comment]
Its more like Torture Pipe completely not affect Radiance

**Comment by Draze22** [comment]
Good catch with Radiance, however items like Cloak of Flames still have to be re-equipped to be affected by Torture Pipe, is this something we can fix?

**Comment by ZLOY5** [comment]
Not really, for that we need refresh modifiers on all affected units. This issue work only for units that was affected by burning before you equip Torture Pipe, if they go out of range and come back damage will be proper.
